common/src/main/scala/org/apache/celeborn/common/metrics/source/Source.scala

```diff
@@ -29,3 +29,8 @@ trait Source {
   def getMetrics: String
   def destroy(): Unit
 }
+
+object Source {
+  val MASTER = "master"
```

Comment:
It looks like these fields are not used. I think they can be removed.

Reply:
thanks, removed